components/InputPreview.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+export default class InputPreview extends Component {
+ render () {
+   return (
+     <div>
+       <input
+        type="text"
+        value={this.props.value}
+        onChange={e => this.props.onChange(e.target.value)}
```

Comment:
Tất cả các Props a đều cần khai báo props-type cho nó và khởi tạo default-value khi cần

Reply:
A nên sửa thành onChange={this.props.onChange} và xử lý get e.target.value ở action sau (chi tiết e sẽ giải thích cụ thể với a sau)

Reply:
done